pageserver/src/tenant.rs

```diff
@@ -479,18 +502,6 @@ impl Tenant {
             "Timeline has no ancestor and no layer files"
         );
```

Comment:
I think you initially wanted that particular check. Then it was left around, seemed to work.

If and when we'll no longer do initdb locally but on compute, this will stop holding. Re: #2592.

Feels like we should break the timeline if this happens, unsure if we are doing that now.

Reply:
I'll just resolve this.